### ISSUE TITLE
Water level sensor does not update

### ISSUE THREAD

**Issue by Maxtropos** [And3rsL/Deebot-for-Home-Assistant#196]
### Is there an existing issue for this?

- [X] I have searched the existing issues and no issue is describing my issue

### The problem

The water level sensor "sensor.ROBOTNAME_water_level " does not update. It sets correctly once when starting hassio. I tried setting the water level by app or by vacuum.send_command, same problem in both ways.
The `waterAmount` value in the log changes correctly, but the sensor does not.

binary_sensor.ROBOTNAME_mop_attached works flawlessly.

Best regards

### On which deebot vacuum you have the issue?

Deebot Ozmo T8+

### Which version of the Deebot component are you using?

3.1.3

### Which version of Home Assistant are you using?

2021.9.7

### What type of installation are you running?

Home Assistant OS

### Country

de

### Continent

eu

### Instance type

Cloud

### Anything in the logs that might be useful for us?
This is the log that appears, when changing the water level in the app.
```

2021-09-29 18:08:43 DEBUG (MainThread) [deebotozmo.ecovacs_mqtt] Got message: topic=iot/atr/onFwBuryPoint/cbc57312-0204-470e-8386-11afea407300/fqxoiu/vLrh/j; payload=b'{"header":{"pri":1,"tzm":480,"ts":"1632931721139","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"content":"{\\"rn\\":\\"bd_setting\\",\\"d\\":{\\"header\\":{\\"pri\\":1,\\"tzm\\":480,\\"ts\\":\\"1632931721140\\",\\"ver\\":\\"0.0.1\\",\\"fwVer\\":\\"1.7.1\\",\\"hwVer\\":\\"0.1.1\\"},\\"body\\":{\\"data\\":{\\"d_id\\":\\"44530\\",\\"d_sum\\":1,\\"d_seq\\":0,\\"d_val\\":\\"{\\\\\\"gid\\\\\\":\\\\\\"G1632411285180\\\\\\",\\\\\\"index\\\\\\":\\\\\\"0000001496\\\\\\",\\\\\\"simpleMode\\\\\\":0,\\\\\\"AI\\\\\\":1,\\\\\\"isPressurized\\\\\\":1,\\\\\\"multiMap\\\\\\":0,\\\\\\"fanspeed\\\\\\":0,\\\\\\"continue\\\\\\":1,\\\\\\"waterAmount\\\\\\":2,\\\\\\"DND\\\\\\":0,\\\\\\"mopremind\\\\\\":1,\\\\\\"mopmode\\\\\\":1,\\\\\\"occurrenceTime\\\\\\":\\\\\\"1632931721113\\\\\\"}\\"}}}}"}}}';
2021-09-29 18:08:43 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle onFwBuryPoint: {'header': {'pri': 1, 'tzm': 480, 'ts': '1632931721139', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632931721140","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"44530","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001496\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":0,\\"continue\\":1,\\"waterAmount\\":2,\\"DND\\":0,\\"mopremind\\":1,\\"mopmode\\":1,\\"occurrenceTime\\":\\"1632931721113\\"}"}}}}'}}}
2021-09-29 18:08:43 DEBUG (MainThread) [deebotozmo.vacuum_bot] Unknown event: fwburypoint with {'header': {'pri': 1, 'tzm': 480, 'ts': '1632931721139', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632931721140","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"44530","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001496\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":0,\\"continue\\":1,\\"waterAmount\\":2,\\"DND\\":0,\\"mopremind\\":1,\\"mopmode\\":1,\\"occurrenceTime\\":\\"1632931721113\\"}"}}}}'}}}

```

**Comment by edenhaus** [comment]
Can you please share the logs, where you see the `waterAmount` changes correctly?

**Comment by Maxtropos** [comment]
Switching from low to medium:
```
2021-09-30 00:32:18 DEBUG (MainThread) [custom_components.deebot.vacuum] async_send_command set_water with {'amount': 'medium'}
2021-09-30 00:32:18 DEBUG (MainThread) [deebotozmo.ecovacs_json] Calling https://portal-eu.ecouser.net/api/iot/devmanager.do with {'auth': '[REMOVED]', 'cmdName': 'setWaterInfo', 'payload': {'header': {'pri': '1', 'ts': 1632954738.591861, 'tzm': 480, 'ver': '0.0.50'}, 'body': {'data': {'amount': 2, 'enable': 0}}}, 'payloadType': 'j', 'td': 'q', 'toId': 'cbc57312-0204-470e-8386-11afea407300', 'toRes': 'vLrh', 'toType': 'fqxoiu'}
2021-09-30 00:32:18 DEBUG (MainThread) [deebotozmo.ecovacs_json] Got {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954736842', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': 'nkqY'}
2021-09-30 00:32:18 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle setWaterInfo: {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954736842', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': 'nkqY'}
2021-09-30 00:32:18 DEBUG (MainThread) [deebotozmo.ecovacs_mqtt] Got message: topic=iot/atr/onFwBuryPoint/cbc57312-0204-470e-8386-11afea407300/fqxoiu/vLrh/j; payload=b'{"header":{"pri":1,"tzm":480,"ts":"1632954736889","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"content":"{\\"rn\\":\\"bd_setting\\",\\"d\\":{\\"header\\":{\\"pri\\":1,\\"tzm\\":480,\\"ts\\":\\"1632954736889\\",\\"ver\\":\\"0.0.1\\",\\"fwVer\\":\\"1.7.1\\",\\"hwVer\\":\\"0.1.1\\"},\\"body\\":{\\"data\\":{\\"d_id\\":\\"965965\\",\\"d_sum\\":1,\\"d_seq\\":0,\\"d_val\\":\\"{\\\\\\"gid\\\\\\":\\\\\\"G1632411285180\\\\\\",\\\\\\"index\\\\\\":\\\\\\"0000001741\\\\\\",\\\\\\"simpleMode\\\\\\":0,\\\\\\"AI\\\\\\":1,\\\\\\"isPressurized\\\\\\":1,\\\\\\"multiMap\\\\\\":0,\\\\\\"fanspeed\\\\\\":1,\\\\\\"continue\\\\\\":1,\\\\\\"waterAmount\\\\\\":1,\\\\\\"DND\\\\\\":1,\\\\\\"mopremind\\\\\\":1,\\\\\\"mopmode\\\\\\":0,\\\\\\"occurrenceTime\\\\\\":\\\\\\"1632954736860\\\\\\"}\\"}}}}"}}}';
2021-09-30 00:32:18 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle onFwBuryPoint: {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954736889', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954736889","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"965965","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001741\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":1,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954736860\\"}"}}}}'}}}
2021-09-30 00:32:18 DEBUG (MainThread) [deebotozmo.vacuum_bot] Unknown event: fwburypoint with {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954736889', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954736889","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"965965","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001741\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":1,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954736860\\"}"}}}}'}}}
```
Switching from medium to high:
```
2021-09-30 00:33:17 DEBUG (MainThread) [custom_components.deebot.vacuum] async_send_command set_water with {'amount': 'high'}
2021-09-30 00:33:17 DEBUG (MainThread) [deebotozmo.ecovacs_json] Calling https://portal-eu.ecouser.net/api/iot/devmanager.do with {'auth': '[REMOVED]', 'cmdName': 'setWaterInfo', 'payload': {'header': {'pri': '1', 'ts': 1632954797.385319, 'tzm': 480, 'ver': '0.0.50'}, 'body': {'data': {'amount': 3, 'enable': 0}}}, 'payloadType': 'j', 'td': 'q', 'toId': 'cbc57312-0204-470e-8386-11afea407300', 'toRes': 'vLrh', 'toType': 'fqxoiu'}
2021-09-30 00:33:17 DEBUG (MainThread) [deebotozmo.ecovacs_json] Got {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954795707', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': 'lzTD'}
2021-09-30 00:33:17 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle setWaterInfo: {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954795707', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': 'lzTD'}
2021-09-30 00:33:17 DEBUG (MainThread) [deebotozmo.ecovacs_mqtt] Got message: topic=iot/atr/onFwBuryPoint/cbc57312-0204-470e-8386-11afea407300/fqxoiu/vLrh/j; payload=b'{"header":{"pri":1,"tzm":480,"ts":"1632954795740","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"content":"{\\"rn\\":\\"bd_setting\\",\\"d\\":{\\"header\\":{\\"pri\\":1,\\"tzm\\":480,\\"ts\\":\\"1632954795740\\",\\"ver\\":\\"0.0.1\\",\\"fwVer\\":\\"1.7.1\\",\\"hwVer\\":\\"0.1.1\\"},\\"body\\":{\\"data\\":{\\"d_id\\":\\"951200\\",\\"d_sum\\":1,\\"d_seq\\":0,\\"d_val\\":\\"{\\\\\\"gid\\\\\\":\\\\\\"G1632411285180\\\\\\",\\\\\\"index\\\\\\":\\\\\\"0000001742\\\\\\",\\\\\\"simpleMode\\\\\\":0,\\\\\\"AI\\\\\\":1,\\\\\\"isPressurized\\\\\\":1,\\\\\\"multiMap\\\\\\":0,\\\\\\"fanspeed\\\\\\":1,\\\\\\"continue\\\\\\":1,\\\\\\"waterAmount\\\\\\":2,\\\\\\"DND\\\\\\":1,\\\\\\"mopremind\\\\\\":1,\\\\\\"mopmode\\\\\\":0,\\\\\\"occurrenceTime\\\\\\":\\\\\\"1632954795723\\\\\\"}\\"}}}}"}}}';
2021-09-30 00:33:17 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle onFwBuryPoint: {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954795740', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954795740","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"951200","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001742\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":2,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954795723\\"}"}}}}'}}}
2021-09-30 00:33:17 DEBUG (MainThread) [deebotozmo.vacuum_bot] Unknown event: fwburypoint with {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954795740', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954795740","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"951200","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001742\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":2,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954795723\\"}"}}}}'}}}
```
Switching from high to ultrahigh:
```
2021-09-30 00:33:53 DEBUG (MainThread) [custom_components.deebot.vacuum] async_send_command set_water with {'amount': 'ultrahigh'}
2021-09-30 00:33:53 DEBUG (MainThread) [deebotozmo.ecovacs_json] Calling https://portal-eu.ecouser.net/api/iot/devmanager.do with {'auth': '[REMOVED]', 'cmdName': 'setWaterInfo', 'payload': {'header': {'pri': '1', 'ts': 1632954833.778, 'tzm': 480, 'ver': '0.0.50'}, 'body': {'data': {'amount': 4, 'enable': 0}}}, 'payloadType': 'j', 'td': 'q', 'toId': 'cbc57312-0204-470e-8386-11afea407300', 'toRes': 'vLrh', 'toType': 'fqxoiu'}
2021-09-30 00:33:54 DEBUG (MainThread) [deebotozmo.ecovacs_json] Got {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954832160', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': '1Tae'}
2021-09-30 00:33:54 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle setWaterInfo: {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954832160', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': '1Tae'}
2021-09-30 00:33:54 DEBUG (MainThread) [deebotozmo.ecovacs_mqtt] Got message: topic=iot/atr/onFwBuryPoint/cbc57312-0204-470e-8386-11afea407300/fqxoiu/vLrh/j; payload=b'{"header":{"pri":1,"tzm":480,"ts":"1632954832189","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"content":"{\\"rn\\":\\"bd_setting\\",\\"d\\":{\\"header\\":{\\"pri\\":1,\\"tzm\\":480,\\"ts\\":\\"1632954832189\\",\\"ver\\":\\"0.0.1\\",\\"fwVer\\":\\"1.7.1\\",\\"hwVer\\":\\"0.1.1\\"},\\"body\\":{\\"data\\":{\\"d_id\\":\\"757631\\",\\"d_sum\\":1,\\"d_seq\\":0,\\"d_val\\":\\"{\\\\\\"gid\\\\\\":\\\\\\"G1632411285180\\\\\\",\\\\\\"index\\\\\\":\\\\\\"0000001743\\\\\\",\\\\\\"simpleMode\\\\\\":0,\\\\\\"AI\\\\\\":1,\\\\\\"isPressurized\\\\\\":1,\\\\\\"multiMap\\\\\\":0,\\\\\\"fanspeed\\\\\\":1,\\\\\\"continue\\\\\\":1,\\\\\\"waterAmount\\\\\\":3,\\\\\\"DND\\\\\\":1,\\\\\\"mopremind\\\\\\":1,\\\\\\"mopmode\\\\\\":0,\\\\\\"occurrenceTime\\\\\\":\\\\\\"1632954832174\\\\\\"}\\"}}}}"}}}';
2021-09-30 00:33:54 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle onFwBuryPoint: {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954832189', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954832189","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"757631","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001743\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":3,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954832174\\"}"}}}}'}}}
2021-09-30 00:33:54 DEBUG (MainThread) [deebotozmo.vacuum_bot] Unknown event: fwburypoint with {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954832189', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954832189","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"757631","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001743\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":3,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954832174\\"}"}}}}'}}}
```
Switching from ultrahigh to low:
2021-09-30 00:34:22 DEBUG (MainThread) [custom_components.deebot.vacuum] async_send_command set_water with {'amount': 'low'}
2021-09-30 00:34:22 DEBUG (MainThread) [deebotozmo.ecovacs_json] Calling https://portal-eu.ecouser.net/api/iot/devmanager.do with {'auth': '[REMOVED]', 'cmdName': 'setWaterInfo', 'payload': {'header': {'pri': '1', 'ts': 1632954862.286009, 'tzm': 480, 'ver': '0.0.50'}, 'body': {'data': {'amount': 1, 'enable': 0}}}, 'payloadType': 'j', 'td': 'q', 'toId': 'cbc57312-0204-470e-8386-11afea407300', 'toRes': 'vLrh', 'toType': 'fqxoiu'}
2021-09-30 00:34:22 DEBUG (MainThread) [deebotozmo.ecovacs_json] Got {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954860627', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': 'Tjla'}
2021-09-30 00:34:22 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle setWaterInfo: {'ret': 'ok', 'resp': {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954860627', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'code': 0, 'msg': 'ok'}}, 'id': 'Tjla'}
2021-09-30 00:34:23 DEBUG (MainThread) [deebotozmo.ecovacs_mqtt] Got message: topic=iot/atr/onFwBuryPoint/cbc57312-0204-470e-8386-11afea407300/fqxoiu/vLrh/j; payload=b'{"header":{"pri":1,"tzm":480,"ts":"1632954860647","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"content":"{\\"rn\\":\\"bd_setting\\",\\"d\\":{\\"header\\":{\\"pri\\":1,\\"tzm\\":480,\\"ts\\":\\"1632954860648\\",\\"ver\\":\\"0.0.1\\",\\"fwVer\\":\\"1.7.1\\",\\"hwVer\\":\\"0.1.1\\"},\\"body\\":{\\"data\\":{\\"d_id\\":\\"543998\\",\\"d_sum\\":1,\\"d_seq\\":0,\\"d_val\\":\\"{\\\\\\"gid\\\\\\":\\\\\\"G1632411285180\\\\\\",\\\\\\"index\\\\\\":\\\\\\"0000001744\\\\\\",\\\\\\"simpleMode\\\\\\":0,\\\\\\"AI\\\\\\":1,\\\\\\"isPressurized\\\\\\":1,\\\\\\"multiMap\\\\\\":0,\\\\\\"fanspeed\\\\\\":1,\\\\\\"continue\\\\\\":1,\\\\\\"waterAmount\\\\\\":0,\\\\\\"DND\\\\\\":1,\\\\\\"mopremind\\\\\\":1,\\\\\\"mopmode\\\\\\":0,\\\\\\"occurrenceTime\\\\\\":\\\\\\"1632954860643\\\\\\"}\\"}}}}"}}}';
2021-09-30 00:34:23 DEBUG (MainThread) [deebotozmo.vacuum_bot] Handle onFwBuryPoint: {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954860647', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954860648","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"543998","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001744\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":0,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954860643\\"}"}}}}'}}}
2021-09-30 00:34:23 DEBUG (MainThread) [deebotozmo.vacuum_bot] Unknown event: fwburypoint with {'header': {'pri': 1, 'tzm': 480, 'ts': '1632954860647', 'ver': '0.0.1', 'fwVer': '1.7.1', 'hwVer': '0.1.1'}, 'body': {'data': {'content': '{"rn":"bd_setting","d":{"header":{"pri":1,"tzm":480,"ts":"1632954860648","ver":"0.0.1","fwVer":"1.7.1","hwVer":"0.1.1"},"body":{"data":{"d_id":"543998","d_sum":1,"d_seq":0,"d_val":"{\\"gid\\":\\"G1632411285180\\",\\"index\\":\\"0000001744\\",\\"simpleMode\\":0,\\"AI\\":1,\\"isPressurized\\":1,\\"multiMap\\":0,\\"fanspeed\\":1,\\"continue\\":1,\\"waterAmount\\":0,\\"DND\\":1,\\"mopremind\\":1,\\"mopmode\\":0,\\"occurrenceTime\\":\\"1632954860643\\"}"}}}}'}}}

**Comment by edenhaus** [comment]
I released [3.1.4b2](https://github.com/And3rsL/Deebot-for-Home-Assistant/releases/tag/3.1.4b2), which should solve this issue.
Also I have implemented a way that, we should get informed, when the settings are changed outside from HA (ex. app).
Finger crossed, I hope it will work.
Can you please test it? You can install the beta via hacs

Are you using the ecovacs server or bumper?

**Comment by Maxtropos** [comment]
> I released [3.1.4b2](https://github.com/And3rsL/Deebot-for-Home-Assistant/releases/tag/3.1.4b2), which should solve this issue. Also I have implemented a way that, we should get informed, when the settings are changed outside from HA (ex. app). Finger crossed, I hope it will work. Can you please test it? You can install the beta via hacs
> 
> Are you using the ecovacs server or bumper?

I tried 3.1.4B3, the issue has been fixed. Works flawlessly now.
I am using the ecovacs server.
